jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -73,7 +73,7 @@ node ()  {
 	def domain = params.domain
 
 	if(params.domain && params.domain.trim() != ""){
-		if(params.domain != "platform"){
+		if(params.domain != "platform" && configLoader.SCM.TYPE != "gitlab" ){
```

Comment:
Same as above - Why is this required?

Reply:
This is because the diversity in the repo naming of platform services and user services.
In Gitlab the same webhook trigger job is using for platform services and user services.
So this if block is needed.